src/github3/users.py

```diff
@@ -144,13 +241,20 @@ class Email(models.GitHubCore):
 
         A boolean value representing whether the address is the primary
         address for the user or not
+
+    .. attribute:: visibility
+
+        A string value representing whether an authenticated user can view the
+        email address. Use ``public`` to allow it, ``private`` to disallow it.
     """
 
     def _update_attributes(self, email):
         self.email = email['email']
         self.verified = email['verified']
-        self.primary = email['primary']
-        self.visibility = email['visibility']
+        if 'primary' in email:
+            self.primary = email['primary']
+        if 'visibility' in email:
+            self.visibility = email['visibility']
```

Comment:
The emails returned by the GPG key API only have the `email` and the `verified` attributes (see: https://developer.github.com/v3/users/gpg_keys/#response-2). An alternative implementation is to split the `Email` model into `ShortEmail` and `Email`; I chose the above solution for simplicity, but I'm open for suggestions!

Reply:
I'd rather consistency over simplicity. That means creating ShortEmail and Email classes.